### PR TITLE
Feature: Contact Types

### PR DESCRIPTION
**Pull request by krak3n**
This PR implements all the base types required for marshalling and unmarshmaling XML responses for Contacts.

Sorry this is a rather large & boring one :wink:

### CODE REVIEW

File: contacts.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+//      </PaymentTerms>
+//   </Contact>
+type Contact struct {
+	ValidationErrors // Used for validating POST/PUT requests
+
+	// The following can be set on POST/PUT requests
+	ContactID                 string          `xml:"ContactID,omitempty`
+	ContactNumber             string          `xml:"ContactNumber,omitempty`
+	AccountNumber             string          `xml:"AccountNumber,omitempty"`
+	ContactStatusa            string          `xml:"ContactStatus,omitempty"`
```
Comment (gregatsoon) [inline contacts.go]: Is this right? Should it be `ContactStatus` (no `a`)

Comment (krak3n) [reply contacts.go]: Nice catch 👍

**Comment by gregatsoon** [comment]
That can't have been fun. Looking good 👍

**Review by jamesjwarren — APPROVED** [review]
Nice one! 👍

**Pull request merged**